zallet/src/components/json_rpc.rs

```diff
@@ -0,0 +1,7 @@
+//! JSON-RPC endpoint.
+//!
+//! This provides JSON-RPC methods that are (mostly) compatible with the `zcashd` wallet
```

Comment:
```suggestion
//! This provides JSON-RPC methods that are mostly compatible with the `zcashd` wallet
```
We are not trying for full compatibility, I think.

Reply:
What I was meaning here is that some JSON-RPC methods with the same name will be exactly compatible with their equivalent `zcashd` methods, while others may have deviations.